testing/cli/test_cli_sct_analyze_lesion.py

```diff
@@ -41,12 +41,38 @@ def dummy_lesion(request, tmp_path):
 
 
 def compute_expected_measurements(dim, starting_coord=None, path_seg=None):
-    # Find the minimum SC area surrounding the lesion
     if path_seg:
+        # Find the minimum SC area surrounding the lesion
         data_seg = Image(path_seg).data
         min_area = min(np.sum(data_seg[:, n_slice, :])
                        for n_slice in range(starting_coord[1], starting_coord[1] + dim[1]))
 
+        # Find the minimum mid-sagittal tissue bridge width for each LR slice in the lesion
+        x = starting_coord[0] + (dim[0] // 2)  # Compute midpoint of lesion (to split into dorsal/ventral regions)
+        tissue_bridges = {}
+        for z in range(starting_coord[2], starting_coord[2] + dim[2]):
+
+            # for each SI slice in the lesion, compute the bridge widths
+            dorsal_bridge_widths, ventral_bridge_widths = [], []
+            for y in range(starting_coord[1], starting_coord[1] + dim[1]):
+                # compute ventral widths
+                ventral_sc_width = np.sum(data_seg[:x, y, z])
+                ventral_lesion_width = (x - starting_coord[0])
+                ventral_bridge_widths.append(max(0.0, ventral_sc_width - ventral_lesion_width))
+                # compute dorsal widths
+                dorsal_sc_width = np.sum(data_seg[x:, y, z])
+                dorsal_lesion_width = (starting_coord[0] + dim[0] - x)
+                dorsal_bridge_widths.append(max(0.0, dorsal_sc_width - dorsal_lesion_width))
+
+            # find minimum widths
+            # FIXME: Because t2.nii.gz starts as AIL but is reoriented to RPI, there is a RL flip, meaning the
+            #        z slice number of the AIL image won't correspond to the z slice number used for the column name.
+            z_flipped = data_seg.shape[2] - (z + 1)  # account for 0-based index
+            tissue_bridges[f"slice_{z_flipped}_dorsal_bridge_width [mm]"] = min(dorsal_bridge_widths)
+            tissue_bridges[f"slice_{z_flipped}_ventral_bridge_width [mm]"] = min(ventral_bridge_widths)
+            tissue_bridges[f"slice_{z_flipped}_total_bridge_width [mm]"] = (min(dorsal_bridge_widths) +
+                                                                            min(ventral_bridge_widths))
```

Comment:
I also noticed that the "total" value is the sum of the two (separate) ventral/dorsal minimums.

However, if the ventral minimum and the dorsal minimum are on different axial slices, then the computed value will be a different value than "slicewise minimum total width" (i.e. computing, for each slice, the full SC width minus the full lesion width).

Is this intentional?

Reply:
Good point!

> Is summing the two separate minimums intentional?

Yes, this is intentional. I got inspiration for this from a SOP for manual measurements (shown below). The ventral bridge (VB) can be located in a different axial slice than the dorsal bridge (DB). `total_bridge` is than just a simple sum of VB and DB.

<img width="250" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/5a02bc2f-14ca-4185-bd10-e711a2dd0654">